paramak/parametric_components/port_cutters_circular.py

```diff
@@ -7,10 +7,9 @@ class PortCutterCircular(ExtrudeCircleShape):
     other components (eg. blanket, vessel,..) in order to create ports.
 
     Args:
-        z_pos (float): Z position (cm) of the port
-        height (float): height (cm) of the port
-        width (float): width (cm) of the port
-        distance (float): extruded distance (cm) of the cutter
+        center_point (tuple of floats): Defaults to (0, 0).
```

Comment:
```suggestion
        center_point ((float, float)): Defaults to (0, 0).
```

I think that for that type of argument this is the correct syntax.